examples/message-ubench/main.pony

```diff
@@ -278,12 +288,20 @@ actor Pinger
 class Tick is TimerNotify
   let _env: Env
   let _sync_leader: SyncLeader
+  let _report_count: U64
+  var _tick_count: U64 = 0
 
-  new iso create(env: Env, sync_leader: SyncLeader) =>
+  new iso create(env: Env, sync_leader: SyncLeader, report_count: U64) =>
     _env = env
     _sync_leader = sync_leader
+    _report_count = report_count
 
     fun ref apply(timer: Timer, count: U64): Bool =>
-      _sync_leader.tick_fired(count)
-      true
-
+      _tick_count = _tick_count + count
+      let done = (_report_count > 0) and (_tick_count >= _report_count)
+      _sync_leader.tick_fired(done)
+      if done then
```

Comment:
`not(done)` instead of `if done then false else true end`?

Reply:
of course, txs